contracts/presets/ERC20PresetFixedSupply.sol

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.6.2;
+
+import "../token/ERC20/ERC20.sol";
+import "../token/ERC20/ERC20Burnable.sol";
+
+/**
+ * @dev Extension of {ERC20} and {ERC20Burnable} with fixed initialSupply.
+ *
+ * Tip: Contract is preconfigured with fixed initial supply.
+ * minting / pausing functionality is not supported. This removes the need of access control and thereby governance. 
+ */
+contract ERC20PresetFixedSupply is ERC20, ERC20Burnable {
+    constructor(
+        string memory name,
+        string memory symbol,
+        uint256 initialSupply
+    ) public ERC20(name, symbol) {
+        _mint(msg.sender, initialSupply);
```

Comment:
I think it would be better to use an address in the constructor arguments instead of `msg.sender`. This improves "composability", for example it allows you to use a generic factory to deploy the contract and assign the initial balance to a separate account.

In hindsight this may be an issue with the current roles presets, since they assign roles to the deployer account.

Reply:
Got it! Replacing msg.sender by an argument 'owner' of type address.